metering/metering-installing-metering.adoc

```diff
@@ -5,41 +5,41 @@ include::modules/common-attributes.adoc[]
 
 include::modules/metering-install-operator.adoc[leveloffset=+1]
 
 // This content is included directly in the assembly and NOT in a module because the workflow for installing metering requires linking off to the config docs, and xrefs are not allowed in modules. [klamenzo]
 [id="metering-install-metering-stack_{context}"]
 == Installing the Metering stack
 
-To install the metering stack, create a `MeteringConfig` resource and add your configuration details.
+To install the metering stack, first review the xref:../metering/configuring-metering/metering-about-configuring.adoc#metering-about-configuring[configuration options] and create a `MeteringConfig` resource. You can begin the process below to generate a default `MeteringConfig`, then use the examples in the documentation to modify this default file for your specific installation. 
 
-Before creating your `MeteringConfig`:
+Review the following topics to create your `MeteringConfig`:
 
-* Review the installation options in xref:../metering/metering-about-installing-metering.adoc#metering-about-install_about-installing-metering[About installing metering]
+* For prerequisites and general requirements, review the installation options in xref:../metering/metering-about-installing-metering.adoc#metering-about-install_about-installing-metering[About installing metering]
+* For configuration options, review xref:../metering/configuring-metering/metering-about-configuring.adoc#metering-about-configuring[About configuring metering]. 
 * At a minimum, you need to xref:../metering/configuring-metering/metering-configure-persistent-storage.adoc#metering-configure-persistent-storage[configure persistent storage] and xref:../metering/configuring-metering/metering-configure-hive-metastore.adoc#metering-configure-hive-metastore[configure the Hive metastore].
 
-After entering your configuration details, the `MeteringConfig` resource installs all the necessary metering components inside the openshift-metering namespace.
-
 [IMPORTANT]
 ====
 There can only be one `MeteringConfig` resource in the `openshift-metering` namespace. Any other configuration is not supported.
 ====
 
 .Procedure
 
-. From the web console, ensure you are on the Operator Details page for the Metering Operator in the openshift-metering project. You can navigate to this page by clicking *Operators* -> *Installed Operators*, then selecting the Metering Operator.
+. From the web console, ensure you are on the Operator Details page for the Metering Operator in the `openshift-metering` project. You can navigate to this page by clicking *Operators* -> *Installed Operators*, then selecting the Metering Operator.
```

Comment:
s/Operator Details/*Operator Details* to match other instances:
https://metering-updates-for-install-workflow--ocpdocs.netlify.com/openshift-enterprise/latest/metering/metering-installing-metering.html#metering-install-operator_installing-metering